book/src/spec.md

```diff
@@ -0,0 +1,628 @@
+Physical stream specification
+=============================
```

Comment:
I personally really dislike the ### syntax, because it's so much harder to make out which level a header is, and to distinguish the headers from the text in the first place. Unfortunately, as far as I know, there is no clearer syntax for h3 and beyond, so I'm forced to use them there.

Ultimately I don't see why this matters at all, I don't think it's anyone's intention to have people read raw markdown in the first place? I just find this syntax more convenient to write.